recipe/meta.yaml

```diff
@@ -1,17 +1,18 @@
 {% set name = "requests" %}
-{% set version = "2.27.1" %}
-
+{% set version = "2.28.0" %}
+{% set hash = "d568723a7ebd25875d8d1eaf5dfa068cd2fc8194b2e483d7b1f7c81918dbec6b" %}
+{% set build_number = "0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/requests-{{ version }}.tar.gz
-  sha256: 68d7c56fd5a8999887728ef304a6d12edc7be74f1cfa47714fc8b414525c9a61
+  sha256: {{ hash }}
 
 build:
-  number: 0
+  number: {{ build_number }}
   noarch: python
```

Comment:
We are currently not building noarch packages
https://github.com/AnacondaRecipes/requests-feedstock/blob/2.28.0/recipe/meta.yaml#L16

need to also add the skip section  and modify the `python` pinings

Reply:
When we remove `noarch:python` and make an arch-specific package we can use `skip: True  # [py<36]`, and also remove pinning for `python` in `host` and `run`.